src/js/views/patients/widgets/widgets.js

```diff
@@ -34,14 +40,10 @@ const widgets = {
   widget: View.extend({
     className: 'widgets-value',
     initialize() {
-      const slug = this.getOption('slug');
-      const widgetValues = Radio.request('entities', 'get:widgetValues:model', slug, this.model.id);
-      this.values = widgetValues.get('values');
```

Comment:
values was only being attached here, but now it's up in `buildWidget` so if it works for one it works for all now